packages/documentation/gatsby-config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   siteMetadata: {
     description: `Resources and documention for the Development within the VA.gov project`,
     siteUrl: `https://department-of-veterans-affairs.github.io/veteran-facing-services-tools`,
-    title: `VA.gov Developer's Documentations`,
+    title: `VA.gov Developer's Documentation`,
```

Comment:
Could we match this to the title in the site header? "Client Application Documentation"